release_notes/ocp-4-14-release-notes.adoc

```diff
@@ -2121,7 +2121,7 @@ In the following tables, features are marked with the following statuses:
 |Egress IPs on additional network interfaces
 |Not Available
 |Not Available
-|Technology Preview
+|General Availability
```

Comment:
@stevsmit and @JoeAldinger At Kathryn's request I am going to merge this. However, either in a separate PR or possibly a RN clean up PR, please remove this feature from the Tech Preview table. It is no longer necessary to reference it here, seeing as the feature is going straight to GA.

Reply:
@mjpytlak cleaned up in [this](https://github.com/openshift/openshift-docs/pull/67048) PR